packages/core/lib/debug/interpreter.js

```diff
@@ -628,7 +634,13 @@ class DebugInterpreter {
       case "l":
         if (this.session.view(session.status.loaded)) {
           this.printer.printFile();
-          this.printer.printState(LINES_BEFORE_LONG, LINES_AFTER_LONG);
```

Comment:
These two constants are used in the printErrorLocation() call in case s.

Reply:
Oh, I see!  I missed that, thanks.  That raises the question: Should that case, or `printErrorLocation`, make use of the settings you're introducing?  I think, if they're going to be saved, it probably should be.  (And if they're not going to be saved, maybe `s` should also get these options?  It could share the behavior with`l`, since they're both about printing out source code.)